src/shared/directives/conditional-show/conditional-show-form.directive.ts

```diff
@@ -109,25 +109,25 @@ export class ConditionalShowFormDirective implements OnInit, AfterViewInit, OnDe
   }
 
   // make sure for the 3 callbacks that we are bound to this via an arrow function
-  private handleFormControl = (formControl: FormControl): void => {
```

Comment:
Again, nothing has actually changed in here so I can't see where the conflict would be and this simply overwrites what was recently changed in `master`, which included an indentation fix. I personally prefer `c: FormControl` over `formControl: FormControl` (I think I even did that) but this was changed back in `master`.. so this seems like a pointless overwrite.